src/utils/filterTodos.ts

```diff
@@ -0,0 +1,21 @@
+import { Todo } from '../types/Todo';
+import { TodoCompletedCategory } from '../types/todoCompletedCategory';
+
+export function filterTodos(
+  todos: Todo[],
+  todoCategory: TodoCompletedCategory,
+  query: string,
+): Todo[] {
+  switch (todoCategory) {
+    case TodoCompletedCategory.active:
+      return todos.filter(
+        todo => !todo.completed && todo.title.toLowerCase().includes(query),
+      );
+    case TodoCompletedCategory.competed:
```

Comment:
There is a typo in the case `TodoCompletedCategory.competed`. It should be `TodoCompletedCategory.completed` to match the corrected enum value.